### DISCUSSION TITLE
Best Way To Get Two Window Size Options From One Keypress

### DISCUSSION THREAD

**Discussion by peterinjapan** [rxhanson/RectanglePro-Community#587] [General]
I'm a big fan of Rectangles Pro, and have found the creator to be very helpful with past issues I've had with the program.

One thing I'm trying to figure out is, how to get a single keypress that can set a window to left half/right half, but if I click it again it will toggle additional sizes. usually I have my screens sorted as left half/right half, using this setting (so that each half is slightly more than 50% on the left and right side).

![Screenshot 2024-11-14 at 13 27 52](https://github.com/user-attachments/assets/719010ba-6399-40fc-b9ea-96cccbf7dfef)

What I want, though, is for a second (or a third) click of the key (control + right/left arrow) to switch to a slightly different size, like the left or right 1/3 of the screen, because I need a smaller size. Can any suggest a way to get "toggle size 1-2-3" with repeated keypresses? I can't seem to figure out a good way.

**Comment by rxhanson** [comment]
Here's a slightly older, user-created tutorial for adding repeat commands to custom size & position: https://github.com/rxhanson/RectanglePro-Community/discussions/5

 In short, you mouse over the size/position that you've already created and hit the + for new repetition.
